packages/cdk8s-plus-17/src/job.ts

```diff
@@ -13,6 +13,40 @@ import { Volume } from './volume';
  */
 export interface JobProps extends ResourceProps, PodTemplateProps {
 
+  /**
+   * Specifies the duration in seconds relative to the startTime that
+   * the job may be active before the system tries to terminate it.
+   *
+   * @default - If unset, then there is no deadline.
+   */
+  readonly activeDeadline?: Duration;
+
+  /**
+   * Specifies the number of retries before marking this job failed.
+   *
+   * @default - If not set, system defaults to 6.
+   */
+  readonly backoffLimit?: number;
+
+  /**
+   * Specifies the desired number of successfully finished pods the job should be run with.
+   * Setting to 1 means that parallelism is limited to 1
+   * and the success of that pod signals the success of the job.
+   *
+   * @default - If not set, success of any pod signals the success of all pods.
+   */
+  readonly completions?: number;
+
+  /**
+   * Specifies the maximum desired number of pods the job should run at any given time.
+   * The actual number of pods running in steady state will be less than this number when
+   * ((.spec.completions - .status.successful) < .spec.parallelism),
+   * i.e. when the work left to do is less than max parallelism.
+   *
+   * @defaul - If unset, there will be no parallelism.
+   */
+  readonly parallelism?: number;
```

Comment:
If there is no parallelism, is there meaning to the `completions` property? i.e does specifying 2 completions with no parallelism makes sense? 

Im wondering what the relationship is between these two properties and if we can somehow combine them to something less error prone.